charts/kcp/templates/kcp.yaml

```diff
@@ -213,8 +224,8 @@ spec:
         - --oidc-client-id={{ .Values.oidc.clientId }}
         - --oidc-groups-claim={{ .Values.oidc.groupClaim }}
         - --oidc-username-claim={{ .Values.oidc.usernameClaim }}
-        - "--oidc-username-prefix=rh-sso:"
-        - "--oidc-groups-prefix=rh-sso:"
+        - "--oidc-username-prefix={{ .Values.oidc.usernamePrefix }}"
+        - "--oidc-groups-prefix={{ .Values.oidc.groupsPrefix }}:"
```

Comment:
We definitely need to consistently apply the delimiter at the end of the prefix.  Whether to add the colon or not, I lean towards not. This will allow the user to choose their preferred delimiter.